cli/man/splinter-circuit-propose.1.md

```diff
@@ -123,14 +123,15 @@ EXAMPLES
 This command proposes a simple circuit with one other node.
 
 * The proposing node has ID `alpha001` and endpoint `tls://splinterd-node-acme001:8044`.
-* The other node has ID `beta001` and endpoint `tls://splinterd-node-beta001:8044`.
+* The other node has ID `beta001` and endpoints `tls://splinterd-node-beta001:8044`
+  and `wss://splinterd-node-beta001:8045`.
```

Comment:
Should use tcp:// and tcps:// as examples here since those are implemented and stable.

Reply:
Updated